test/data/datasets/small_test_dataset_with_text.arff

```diff
@@ -0,0 +1,21 @@
+@RELATION small_test_dataset
+
+@ATTRIBUTE missing   numeric
+@ATTRIBUTE cat_int   {0,1,2,3,4,5}
+@ATTRIBUTE cat_str   {T,F}
+@ATTRIBUTE num_int   integer
+@ATTRIBUTE num_float real
+@ATTRIBUTE text_col  string
+@ATTRIBUTE class     {A,B,C,U}
+
+@DATA
+?,5,F,-1,?,    'text',                  U
```

Comment:
I will add some new fields to account for better test cases. As far as adding a text column to the original dataset; I spoke with Brandon about it some time ago and we decided that it would be best to make a new dataset so as not to interfere with the tests of the other metafeatures (or something along those lines).

Reply:
Having a new dataset for new tests means there is no potentail forthe existing tests on the old dataset to break.